scenarios/appsec/test_conf.py

```diff
@@ -42,7 +44,7 @@ def test_waf_timeout(self):
         interfaces.library.assert_no_appsec_event(r)
 
     @missing_feature(context.library <= "ruby@1.0.0")
-    @missing_feature(library="python", reason="secret is not redacted")
+    @missing_feature(context.library < "python@{}".format(PYTHON_RELEASE_GA_1_1))
```

Comment:
You can use f-strings : 

```suggestion
    @missing_feature(context.library < f"python@{PYTHON_RELEASE_GA_1_1}")
```